FirebaseDatabase/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # v7.5.0
 - [added] Implmemented `queryStartingAfterValue` for FirebaseDatabase queries.
+- [added] Implemented `queryEndingBeforeValue` for FirebaseDatabase queries.
```

Comment:
Please mention pagination. You can also merge the two entries if they end up containing mostly duplicate descriptions.